Microsoft.Toolkit.Uwp.Services/Services/Storage/ILocalStorageService.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.Toolkit.Uwp.Services.Services.Storage
+{
+    /// <summary>
+    /// Store data in the Local environment (only on the current device)
+    /// </summary>
+    public interface ILocalStorageService : IStorageService
```

Comment:
I am commonly using C# interface, mainly with a DI solution. So, in case I want both a local and roaming storage service in my project, I need both interfaces.